src/Datadog.Trace/Tagging/WebTags.cs

```diff
@@ -2,37 +2,28 @@
 
 namespace Datadog.Trace.Tagging
 {
-    internal class WebTags : CommonTags
+    internal class WebTags : InstrumentationTags
```

Comment:
I think it would make sense to move `InstrumentationTags` and all of its derived classes (including this and `HttpTags`) to the `Datadog.Trace.ClrProfiler.Managed` library. Was there a reason these were kept in `Datadog.Trace`?

Reply:
It looks like `AspNetCoreTags` inherits from `WebTags` so nevermind this doesn't need to be changed in this PR. Though I think in a separate PR we should move the `AspNetCore` integration to `Datadog.Trace.ClrProfiler.Managed` instead of having it in `Datadog.Trace`. @lucaspimentel do you know what the reasoning was for this placement?

Reply:
I think of `Datadog.Trace.ClrProfiler.Native` and `Datadog.Trace.ClrProfiler.Managed` as the native and managed components of the CLR profiler. The latter is where all the integration "wrapper methods" live.

The ASP.NET Core instrumentation doesn't use an integration wrapper. It listens for DiagnosticSource events and only relies on the CLR profiler for initialization. I was hoping that someday we could support scenarios users enabling the `DiagnosticSource`/`Activity` observers with only a nuget package and no profiler. In that case there's no need for `Datadog.Trace.ClrProfiler.Managed` at all.